npm/webpack-dev-server/src/loader.ts

```diff
@@ -49,7 +50,11 @@ function buildSpecs (projectRoot: string, files: Cypress.Cypress['spec'][] = [])
 }
 
 // Runs the tests inside the iframe
-export default function loader (this: CypressCTWebpackContext) {
+export default function loader (this: CypressCTWebpackContext & LoaderContext<any>) {
```

Comment:
Don't you want `unknown`? CC @lmiller1990

Reply:
Actually we want `<void>` (or `{}` is probably fine), we are intentionally not passing any options to the loader context.